streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/bean/AlertTemplate.java

```diff
@@ -60,7 +60,7 @@ private static AlertTemplate of(Application application) {
 
     if (ExecutionMode.isYarnMode(application.getExecutionMode())) {
       String format = "%s/proxy/%s/";
-      String url = String.format(format, YarnUtils.getRMWebAppURL(), application.getAppId());
+      String url = String.format(format, YarnUtils.getRMWebAppURL(false), application.getAppId());
```

Comment:
do we have to get dynamic url here ? if not, we can omit the false parameter.

Reply:
> do we have to get dynamic url here ? if not, we can omit the false parameter.

If the latest YARN RM url is obtained every time, it will result in more network requests and resource consumption. Obtaining a currently valid active rm url requires a lot of work, as you can see in `org.apache.hadoop.yarn.util.RMHAUtils.findActiveRMHAId` and [streampark](https://github.com/apache/incubator-streampark/blob/827fe32449a25d05a73999a59727f782529e69ff/streampark-common/src/main/scala/org/apache/streampark/common/util/YarnUtils.scala#L159.), by default, we return the previously active rm url. If this url is not available, we will retrieve the latest active node again.